docs/Animations.md

```diff
@@ -7,24 +7,270 @@ permalink: docs/animations.html
+development, but have started to coalesce around two complimentary systems:
+`LayoutAnimation` for animated global layout transactions, and `Animated` for
+more granular and interactive control of specific values.
+
+### Animated ###
+
+The `Animated` library is designed to make it very easy to concisely express a
+wide variety of interesting animation and interaction patterns in a very
+performant way. `Animated` focuses on declarative relationships between inputs
+and outputs, with configurable transforms in between, and simple `start`/`stop`
+methods to control time-based animation execution.  For example, a complete
+component with a simple spring bounce on mount looks like this:
+
+```javascript
+class Playground extends React.Component {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      bounceValue: new Animated.Value(1.5),   // Start large
+    };
+  }
+  render(): ReactElement {
+    return (
+      <Animated.Image                         // Base: Image, Text, View
+        source={{uri: 'http://i.imgur.com/XMKOH81.jpg'}}
+        style={{
+          flex: 1,
+          transform: [                        // `transform` is an ordered array
+            {scale: this.state.bounceValue},  // Map `bounceValue` to `scale`
+          ]
+        }}
+      />
+    );
+  }
+  componentDidMount() {
+    Animated.spring(                          // Base: spring, decay, timing
+      this.state.bounceValue,                 // Animate `bounceValue`
+      {
+        toValue: 0.8,                         // Animate to smaller size
+        friction: 1,                          // Bouncier spring
+      }
+    ).start();                                // Start the animation
+  }
+}
+```
+
+`bounceValue` is initialized to 1 as part of `state` in the constructor, and
+mapped to the scale transform on the image.  Behind the scenes, the numeric
+value is extracted and used to set scale.  When the component mounts, a spring
+animation is started on `bounceValue` which will update all of it's dependent
+mappings on each frame as the spring animates (in this case, just the scale).
+This is done in an optimized why that is faster than calling setState and
+re-rendering, and with future optimizations will be able to be done without
+invoking JavaScript at all since the entire configuration is declarative and can
+be serialized to native which can handle the animation on its own after it's
+started.
+
+Most everything you need hangs directly off the `Animated` module.  This
+includes two value types, `Value` for single values and `ValueXY` for vectors,
+three animation types, `spring`, `decay`, and `timing`, and three component
+types `View`, `Text`, and `Image`.  You can make any other component animated with
+`Animated.createAnimatedComponent`.
+
+The three animation types can be used to create almost any animation curve you
+want because each can be customized:
+
+* `spring`: Simple single-spring physics model that matches Origami.
```

Comment:
[Origami](https://facebook.github.io/origami/)

* probably not everyone is familiar with what this is